### PR TITLE
update comments in without-webhooks server samples

### PR DESCRIPTION
**Pull request by bg-stripe**
r? @adreyfus-stripe

### CODE REVIEW

File: without-webhooks/server/java/src/main/java/com/stripe/sample/Server.java [hunk 1]
```diff
@@ -140,18 +140,20 @@ public static void main(String[] args) {
             try {
                 if (postBody.getPaymentIntentId() == null) {
                     int orderAmount = calculateOrderAmount(postBody.getItems());
-                    // Create new PaymentIntent for the order
+                    // Create new PaymentIntent with a payment method ID from the client.
```
Comment (adreyfus-stripe) [inline without-webhooks/server/java/src/main/java/com/stripe/sample/Server.java]: nit: if we refer to a Stripe resource we should refer to it camelcase like PaymentMethod

**Comment by adreyfus-stripe** [comment]
thank you! feel free to merge after my nit comment

File: without-webhooks/server/java/src/main/java/com/stripe/sample/Server.java (hunk 1 above)
Comment (bg-stripe) [reply without-webhooks/server/java/src/main/java/com/stripe/sample/Server.java]: ack!

**Comment by bg-stripe** [comment]
done! @adreyfus-stripe can you merge (or give me write access 😛 )

**Comment by adreyfus-stripe** [comment]
@bg-stripe How about both! 😆

**Pull request merged**